src/common/matrixproperties.jl

```diff
@@ -0,0 +1,59 @@
+"""
+    isisometry(A; side=:left, isapprox_kwargs...) -> Bool
+
+Test whether a linear map is an isometry, where the type of isometry is controlled by `kind`:
+
+- `side = :left` : `A' * A ≈ I`. 
+- `side = :right` : `A * A` ≈ I`.
```

Comment:
That seems to be the wrong symbol for adjoint:
```suggestion
- `side = :right` : `A * A' ≈ I`.
```